tests/python_client/testcases/test_search.py

```diff
@@ -1810,7 +1808,7 @@ def test_search_max_nq(self, auto_id, dim, _async):
         expected: search successfully with max nq
         """
         self._connect()
-        nq = 17000
+        nq = 16384
```

Comment:
Do we have case for nq=16385? (maximum+1)?

Reply:
> Do we have case for nq=16385? (maximum+1)?

No.

Reply:
> Do we have case for nq=16385? (maximum+1)?

Added in #25317.